### ISSUE TITLE
Make double quotes possible to initialize in markup

### ISSUE THREAD

**Issue by trevorbaca** [Abjad/abjad#431]
Example:

```
>>> Markup('\\"foo''")
  File "<stdin>", line 1
    Markup('\\"foo''")
                     ^
SyntaxError: EOL while scanning string literal
>>> Markup('\\"foo\\"')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/trevorbaca/Documents/abjad/abjad/tools/markuptools/Markup.py", line 97, in __init__
    parsed = lilypondparsertools.LilyPondParser()(to_parse)
  File "/Users/trevorbaca/Documents/abjad/abjad/tools/lilypondparsertools/LilyPondParser.py", line 165, in __call__
    lexer=self._lexer,
  File "/Users/trevorbaca/Documents/abjad/abjad/tools/lilypondparsertools/_parse.py", line 249, in _parse
    tok = self.errorfunc(errtoken)
  File "/Users/trevorbaca/Documents/abjad/abjad/tools/lilypondparsertools/LilyPondSyntacticalDefinition.py", line 3447, in p_error
    raise LilyPondParserError(p)
abjad.tools.exceptiontools.LilyPondParserError.LilyPondParserError: LexToken(\,'\\',1,10)
>>> 
```

**Comment by trevorbaca** [comment]
Still valid in June 2017.

**Comment by trevorbaca** [comment]
I've added an `abjad.Markup.from_literal()` constructor to bypass the parser and handle initialization:
```
>>> markup = abjad.Markup.from_literal('"foo"')
>>> markup
Markup(contents=['"foo"'])
```
But note there are still format-time problems:
```
>>> abjad.show(markup)
Fri Oct  6 12:35:45 2017
GNU LilyPond 2.19.65
Changing working directory to: `/Users/trevorbaca/.abjad/output'
Processing `/Users/trevorbaca/.abjad/output/2268.ly'
Parsing...
/Users/trevorbaca/.abjad/output/2268.ly:12:18: error: EOF found inside string
\markup { \"foo\"
                  }
/Users/trevorbaca/.abjad/output/2268.ly:12:18: error: Unfinished main input
\markup { \"foo\"
                  }
/Users/trevorbaca/.abjad/output/2268.ly:12:18: error: syntax error, unexpected end of input
\markup { \"foo\"
                  }
fatal error: failed files: "/Users/trevorbaca/.abjad/output/2268.ly"
```

**Comment by trevorbaca** [comment]
Format-time problems still present in September 2018:

```
voice = abjad.Voice("c'4 d'4 e'4 f'4")                                          
markup = abjad.Markup.from_literal('"foo"', literal=True)                       
abjad.attach(markup, voice[0])                                                  
abjad.f(voice)                                                                  
\new Voice
{
    c'4
    - \markup { \"foo\" }
    d'4
    e'4
    f'4
}
```

Output:

```
>>> abjad.show(voice)
Tue Sep  4 23:09:51 2018
GNU LilyPond 2.19.82
Changing working directory to: `/Users/trevorbaca/.abjad/output'
Processing `/Users/trevorbaca/.abjad/output/0886.ly'
Parsing...
/Users/trevorbaca/.abjad/output/0886.ly:16:28: error: EOF found inside string
        - \markup { \"foo\"
                            }
/Users/trevorbaca/.abjad/output/0886.ly:16:28: error: Unfinished main input
        - \markup { \"foo\"
                            }
/Users/trevorbaca/.abjad/output/0886.ly:16:28: error: syntax error, unexpected end of input
        - \markup { \"foo\"
                            }
fatal error: failed files: "/Users/trevorbaca/.abjad/output/0886.ly"
```

**Comment by trevorbaca** [comment]
Fixed in Abjad 3.1. Set `literal=True` like this:

```
voice = abjad.Voice("c'4 d'4 e'4 f'4")                                          
markup = abjad.Markup(r'\markup { "foo" }', literal=True)                       
abjad.attach(markup, voice[0])                                                  
abjad.f(voice)                                                                  
\new Voice
{
    c'4
    - \markup { "foo" }
    d'4
    e'4
    f'4
}
```
